### PR TITLE
Feature/ega archive/batch merge 3

### PR DESCRIPTION
**Pull request by blankdots**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [x] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### CODE REVIEW

**Comment by blankdots** [comment]
The changes from this: https://github.com/EGA-archive/LocalEGA/commit/85f4bc7c29b64e65ecfa9178dea24dfcd741bcbc were missing and were needed.

**Comment by blankdots** [comment]
The #29 needs to be merged first also this https://github.com/neicnordic/LocalEGA-db/pull/5 needs to be merged before this and the last commit removed: https://github.com/neicnordic/LocalEGA/pull/30/commits/409b04b25d1f09d712a67b6700b33cce10d11375

**Comment by blankdots** [comment]
Tested here: https://github.com/NBISweden/LocalEGA-helm/pull/70

File: tests/integration/inbox.bats [hunk 1]
```diff
@@ -80,34 +79,59 @@ function teardown() {
 # but the file was removed from the inbox
 # We should receive a message in the error queue
 
-@test "File not found in inbox goes to error" {
+@test "Ingestion of ghost file goes to error" {
```
Comment (jbygdell) [inline tests/integration/inbox.bats]:
```suggestion
@test "Ingestion of missing file goes to error" {
```

File: tests/integration/inbox.bats [hunk 2]
```diff
@@ -80,34 +79,59 @@ function teardown() {
...
     legarun ${MQ_PUBLISH} --correlation_id ${CORRELATION_ID} files "$MESSAGE"
     [ "$status" -eq 0 ]
 
     # Check that a message with the above correlation id arrived in the expected queue
     # Waiting 20 seconds.
-    retry_until 0 10 2 ${MQ_GET} v1.files.error "${TESTUSER}" "/${TESTFILE}.c4ga"
+    retry_until 0 10 2 ${MQ_FIND} v1.files.error "${CORRELATION_ID}"
     [ "$status" -eq 0 ]
 
 }
```
Comment (jbygdell) [inline tests/integration/inbox.bats]: Either this is a duplication or it needs to be reworked as it seems to be a duplication of the step above.

**Review by jbygdell — DISMISSED** [review]

File: tests/integration/inbox.bats (hunk 2 above)
Comment (blankdots) [reply tests/integration/inbox.bats]: I think it is a different test but tests the same thing as the above, the difference is that in the first test it removes the uploaded file, and in the second tries to ingest a file that does not exist.
I see it here as well: https://github.com/EGA-archive/LocalEGA/blob/master/tests/integration/inbox.bats and it is something we might want to push back upstream

File: tests/integration/inbox.bats (hunk 1 above)
Comment (blankdots) [reply tests/integration/inbox.bats]: done.

File: tests/integration/inbox.bats (hunk 2 above)
Comment (blankdots) [reply tests/integration/inbox.bats]: done.

**Review by jbygdell — APPROVED** [review]

**Review by viklund — APPROVED** [review]

**Pull request merged**